### ISSUE TITLE
SubGroups

### ISSUE THREAD

**Issue by itelo24** [Ylianst/MeshCentral#4269]
Hello,
is it planned or maybe already possible to create subgroups?

**Comment by Ylianst** [comment]
I take it the original request was for sub-groups of devices? Not users?

**Comment by si458** [comment]
> I take it the original request was for sub-groups of devices? Not users?

I believe so yes, for example

site 1 -> front office -> comp1,comp2

site 1 -> back office -> comp3, comp4

**Comment by Ylianst** [comment]
I probably should do this, but it's going to be a massive amount of work. Not looking forward to it.

**Comment by silversword411** [comment]
SC just let's you set multiple database fields...then just sort the gui based on standard sort parameters. Those that want group/subgroups etc learn the rules, and label appropriately. It's a #WinWin

**Comment by ClickLeft** [comment]
> SC just let's you set multiple database fields...then just sort the gui based on standard sort parameters. Those that want group/subgroups etc learn the rules, and label appropriately. It's a #WinWin

Yeah, SC handles this well, and that might be easier to implement :)

**Comment by Ylianst** [comment]
What is SC?

In MeshCentral you can tag devices with one or more tags:

![image](https://user-images.githubusercontent.com/1319013/179637918-b40445cb-6ed1-4bd6-8f62-aabd2699e3d0.png)

When you can sort by tags on the device view:

![image](https://user-images.githubusercontent.com/1319013/179637962-0cc05156-1700-45ff-9fc1-13756ce60861.png)

**Comment by silversword411** [comment]
screen connect

Here's some examples

![2022-07-19_094156 - brave_session groups](https://user-images.githubusercontent.com/13395348/179783102-dae3381c-4368-4ab9-90ea-eacc9c0af670.png)

How you create those groups

![2022-07-19_094353 - brave_ConnectWise_Control_Remote_Support_Software_-_Brav](https://user-images.githubusercontent.com/13395348/179783148-dd6a29f5-12cc-4fe5-b0b4-d6efcf97c8a5.png)

Here's the full library of ways to create groups
https://docs.connectwise.com/ConnectWise_Control_Documentation/Get_started/Host_page/Session_groups/Session_groups_filter_reference

**Comment by itelo24** [comment]
The subgroups would make life easier for us, since we currently look after around 50 locations with a wide variety of devices. From the firewall to switches to Windows servers and clients. Since these are schools, we are talking about around 100 servers and around 1000 client PCs.
In addition, schools could then differentiate between student and teacher PCs, for example.

**Comment by itelo24** [comment]
> I probably should do this, but it's going to be a massive amount of work. Not looking forward to it.

What do you think, when or if you could implement subgroups?

**Comment by si458** [comment]
duplicate https://github.com/Ylianst/MeshCentral/issues/1258
